components/pkg-export-docker/src/build.rs

```diff
@@ -99,12 +99,12 @@ impl<'a> BuildSpec<'a> {
             channel: m
                 .value_of("CHANNEL")
                 .map(ChannelIdent::from)
-                .unwrap_or_default(),
+                .unwrap_or_default(), // or configured_value?
```

Comment:
`default` appears to be consistent with current behavior, though an argument could be made to honor `HAB_BLDR_CHANNEL` here.

@fnichol Do you have thoughts either way on this?

Reply:
Would there be a reason to not honor the environment variable where `--channel` is used, in general? Or, rather, to honor it inconsistently where `--channel` is presented as an option?

Reply:
Well, that would be confusing since `--channel` is used in various contexts.

It would be nice if the CLI documented where the env-var overrides applied. Right now, `pkg export docker --help` says:
```
        --base-pkgs-channel <BASE_PKGS_CHANNEL>
            Install base packages from the specified release channel (default: stable)
…
    -c, --channel <CHANNEL>                        Install packages from the specified release channel (default: stable)
```
Based on that, I think this should just be `unwrap_or_default`, but if we want to make this env var overridable (either with `HAB_BLDR_CHANNEL` or with separate env vars for each` we could do that via [clap's env function.](https://docs.rs/clap/2.32.0/clap/struct.Arg.html#method.env).

I suggest we leave that to a follow-up issue if anyone cares to file it.